packages/ember-toucan-form/package.json

```diff
@@ -25,50 +25,77 @@
     "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
     "prepack": "rollup --config"
   },
+  "peerDependencies": {
+    "@crowdstrike/ember-toucan-styles": "^2.0.1",
+    "@ember/test-helpers": "^2.8.1",
+    "@glimmer/tracking": "^1.1.2",
+    "autoprefixer": "^10.0.2",
+    "ember-source": ">=4.8.0",
+    "fractal-page-object": "^0.3.0",
```

Comment:
I guess we will be exposing test helpers that use this in the future, that's why we have it here already, right? Should this be an optional peer dependency? We aren't enforcing its use, are we?

Reply:
Yeah good point.  We are not at the moment. I'll go ahead and remove this.